Source/WebKit/UIProcess/RemoteLayerTree/RemoteLayerTreeDrawingAreaProxy.h

```diff
@@ -56,7 +56,7 @@ class RemoteLayerTreeDrawingAreaProxy : public DrawingAreaProxy {
     TransactionID nextLayerTreeTransactionID() const { return m_pendingLayerTreeTransactionID.next(); }
     TransactionID lastCommittedLayerTreeTransactionID() const { return m_transactionIDForPendingCACommit; }
 
-    virtual void didRefreshDisplay();
+    virtual void didRefreshDisplay(IPC::Connection*);
```

Comment:
It's hard for me to tell what this Connection represents.

Reply:
I'm going with the CompletionHandler approach.